setup.cfg

```diff
@@ -33,6 +33,7 @@ where = src
 
 [options.extras_require]
 test =
+    ci-watson>=0.3.0
```

Comment:
`ci-watson` should not be necessary for this package.  If it is, we should remove it.  This is only needed (really) for `jwst` regression testing access to Artifactory.  We have got sloppy here, and used one or two of the fixtures outside of the regression testing.  We should fix those.

Reply:
This provides the `_jail` fixture, which is used in a handful of tests.

Reply:
Hmm.

We should really just be using `tmp_path` in those tests I think.  `_jail` is just a lazy way to get it, sorta.

Reply:
I'll add an issue to clean this up